gui/choc_WebView.h

```diff
@@ -785,6 +789,9 @@ struct choc::ui::WebView::Pimpl
         {
             auto path = objc::getString (objc::call<id> (e, "charactersIgnoringModifiers"));
             if (path == " ") return !options->enableSpaceToScroll;
+
+            const auto keyCode = objc::call<unsigned short> (e, "keyCode");
+            if (keyCode == 51) return !options->enableBackspaceToNavigate;
```

Comment:
were you seeing this problem on Windows? I tested briefly and it seemed like it was working

Reply:
Are you referring to testing on main or with these changes? These webview changes are mac-specific, and backspace is forwarded to S1 correctly on Windows after the changes in the platform PR.

Reply:
on main, but i'll double check

Reply:
just confirmed that this does seem to work on `main` on Windows. but fine to keep these changes in since it works with your platform changes too 👍

Reply:
Ah nice, thanks for checking. We definitely need these changes for mac to work. I'm guessing backspace isn't used by Edge to navigate, but it is in WebKit. Maybe?